README.md

```diff
@@ -66,13 +66,12 @@
 Click [here](https://github.com/shella12/RentAnItemBackend) to access the back end app that was built with ruby-on-rails
 
 ### Kanban
-- Click [here]() to see the Front End Kanban project
-- Click [here]() to see the Back End Kanban project
+- Click [here](https://github.com/users/shella12/projects/12) to see the Front End Kanban project
+- Click [here](https://github.com/users/shella12/projects/11) to see the Back End Kanban project
```

Comment:
- [ ] Good job adding frontend tests!  However, they are not working on my end:

![image](https://github.com/shella12/RentAnItemApp/assets/57421823/f887cb51-5643-4b3f-8dfc-88b7dcef3912)

Kindly take a look at this issue so your tests can be run. 👍